cloud_build/app_build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright 2020 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Build old angular dart app to generate flutter benchmark dashboard.
+
+cd app
```

Comment:
pushd/popd instead?

Reply:
Any reason to use `pushd/popd` instead of `cd` here? We do not seem to need to keep track of dirs. I have tested and `cd` works.

Reply:
The scripts are very simple right now but if they grow in the future and require jumping in between folders pushd/popd are useful. This is only a nice to have and not a requirement for this PR

Reply:
Sounds good to me. Just added.